src/Markup/Avalonia.Markup.Xaml/AvaloniaXamlLoader.cs

```diff
@@ -139,7 +139,7 @@ public object Load(Uri uri, Uri baseUri = null, object rootInstance = null)
                     {
                         uriString = new Uri(baseUri, uri).AbsoluteUri;
                     }
-                    throw new XamlLoadException("Error loading xaml at " + uriString, e);
+                    throw new XamlLoadException("Error loading xaml at " + uriString + ": " + e.Message, e);
```

Comment:
Also fixed the annoying problem where the XAML exception message didn't actually include the description of what went wrong.